.travis.yml

```diff
@@ -8,11 +8,10 @@ env:
   global:
     - DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     - DOTNET_CLI_TELEMETRY_OPTOUT: 1
-matrix:
+jobs:
```

Comment:
This change the removal of `sudo: required` were based on feedback from the Travis config validator. Apparently, `jobs` is the preferred name for this key now and `sudo: required` is redundant these days.